game-core/src/main/java/games/strategy/engine/lobby/client/ui/LobbyGameTableModel.java

```diff
@@ -154,19 +138,11 @@ public Object getValueAt(final int rowIndex, final int columnIndex) {
       case Comments:
         return description.getComment();
       case Started:
-        return formatBotStartTime(description.getStartDateTime());
+        return description.getFormattedBotStartTime();
       case GUID:
         return gameList.get(rowIndex).getFirst();
       default:
         throw new IllegalStateException("Unknown column:" + column);
```

Comment:
Just wanted to point out that we usually throw `AssertionError` on impossible conditions